### ISSUE TITLE
Test failure in Travis on master branch

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#6994]
We have a test failure in master branch here: 

https://travis-ci.org/publiclab/plots2/jobs/626310949#L3777

It's from this test:

https://github.com/publiclab/plots2/blob/master/test/integration/signup_flow_email_test.rb#L29

```
 FAIL["test_email_error_messages", #<Minitest::Reporters::Suite:0x000055efe2083310 @name="SignUpTest">, 261.438872575]
 test_email_error_messages#SignUpTest (261.44s)
        Expected false to be truthy.
        test/integration/signup_flow_email_test.rb:29:in `block in <class:SignUpTest>'
```

Strangely it didn't fail in the PR that introduced that line:

https://travis-ci.org/publiclab/plots2/builds/624925676

From this PR: https://github.com/publiclab/plots2/pull/6966

**Comment by jywarren** [comment]
Hmm, any ideas on this, @chen-robert? I think it's a test you modified or wrote? Thank you!!

**Comment by cesswairimu** [comment]
This should be now fixed by #6996. Thanks all
